<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Acer-ALG" failures="1" tests="2" name="BlazeDemo Purchase Flow Tests" time="12.486" errors="0" timestamp="2025-09-30T19:30:58 IST">
  <testcase classname="com.testframework.ui_tests.BlazeDemoPurchaseTest" name="intentionallyFailingTest" time="3.683">
    <failure type="java.lang.AssertionError" message="This test is designed to fail to demonstrate screenshot capture.">
      <![CDATA[java.lang.AssertionError: This test is designed to fail to demonstrate screenshot capture.
at com.testframework.ui_tests.BlazeDemoPurchaseTest.intentionallyFailingTest(BlazeDemoPurchaseTest.java:32)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- intentionallyFailingTest -->
  <testcase classname="com.testframework.ui_tests.BlazeDemoPurchaseTest" name="verifyFlightPurchase" time="4.413"/>
</testsuite> <!-- BlazeDemo Purchase Flow Tests -->
